openmdao/docs/theory_manual/total_derivs/total_derivs_theory.rst

```diff
@@ -0,0 +1,227 @@
+Noting that :math:`\partial R/\partial x` and :math:`d y/d x` are both matrices of size :math:`m \times n`, we see that the direct method requires us to solve :math:`n` linear systems in order to compute all columns of :math:`d y/d x`.
+Nothing that :math:`n` is the length of the design vector, :math:`x`, we see that the compute cost of the direct method scales linearly with the number of design variables.
+
+
+Adjoint Method
+****************
+
+The adjoint approach is derived by first substituting the expression for :math:`d y/ d x` back into the original equation for :math:`d f/d x`.
+
+.. math::
+    \frac{d f}{d x} = \frac{\partial F}{\partial x} - \underbrace{\overbrace{\frac{\partial F}{\partial y}}^{1 \times m} \overbrace{\left[\frac{\partial R}{\partial y}\right]^{-1}}^{m \times m}}_{ \psi }\frac{\partial R}{\partial x}.
+
+By grouping :math:`\partial f/\partial y` and :math:`\left[\partial R / \partial y \right]^{-1}` into a single term we get a :math:`(1 \times m)` row vector.
+This row vector, denoted as :math:`\psi`, is called the adjoint vector.
+As long as :math:`\psi` can be computed efficiently, then the total derivative can be computable as
+
+.. math::
+    \frac{d f}{d x} = \frac{\partial F}{\partial x} - \psi \frac{\partial R}{\partial x}.
+
+Through a bit of manipulation, we find that :math:`\psi` solved for via the linear system
+
+.. math::
+    \underbrace{\left[ \frac{\partial R}{\partial y}  \right]^T}_{m \times m} \overbrace{\psi^T}^{m \times 1} =  \underbrace{\frac{\partial f}{\partial y}^T}_{m \times 1},
+
+which gives us :math:`\psi` for the cost of a single linear solve.
+For a problem with more than one output you would perform one linear solve and compute one adjoint vector for each output.
+For example, if you have one objective and two constraints then you would perform 3 linear solves to compute all the total derivatives you needed.
+
+So the compute cost of the adjoint method scales linearly with the number objectives and constraints you have, but is independent of the number of design variables in your problem.
+
+
+
+
+
+
+Unified Derivatives Equations
+-----------------------------
+
+The analytic equations (direct or adjoint) above can be combined with the chain rule to compute total derivatives for any arbitrary model.
+This would achieve the goal of computing total derivatives, knowing only partial derivatives.
+The only problem is that each new model, composed of different combinations of components connected in different ways would require you re-derive a new combination of chain-rule and direct or forward solves to get the totals you needed.
+
+OpenMDAO provides unique functionality that avoids the need to hand compute the specific total derivative equation for every model.
+Practically, what that means is that you never really have to worry about assembling linear systems or applying the chain rule across the different components in your model.
```

Comment:
I would combine this sentence with the next one...
"Practically, what that means is that you never really have to worry about assembling linear systems or applying the chain rule across the different components in your model, because the framework will do this for you automatically."

Reply:
I think that messes up the order of the thoughts. Its important to point out that OpenMDAO does it for you first. 

changed to `To simplify this process OpenMDAO provides functionality that automatically handles all of that work for you, so that you never actually need to assemble linear systems by hand or propogate derivatives with the chain rule.
In order to understand how it automates the task for an arbitrary model, read on!`